Packs/Rapid7InsightVMCloud/Integrations/Rapid7InsightVMCloud/Rapid7InsightVMCloud.py

```diff
@@ -0,0 +1,455 @@
+            response_data = response.get("data")
+            markdown = tableToMarkdown(
+                'This is Required Scan Information',
+                response_data
+            )
+            result = CommandResults(
+                readable_output=markdown,
+                outputs_prefix='Rapid7.InsighVMCloud.Engines',
+                outputs_key_field='id',
+                outputs=response_data
+            )
+            return result
+        else:
+            return_error('no response')
+
+
+def start_scan_command(client: Client, asset_id: str, name: str) -> CommandResults:
+    """
+    Starts a scan.
+
+    Args:
+        client (Client Object): Client object with request.
+        asset_id : The identifiers of the assets to scan.
+        name : The name of the scan.
+
+    """
+
+    args = demisto.args()
+    assets_id = args.get("asset_id")
+    list_id = assets_id.split(',')
+    method = "POST"
+    endpoint = "/v4/integration/scan"
+    data = {
+        "asset_ids": list_id,
+        "name": name
+    }
+
+    response = client.make_request(
+        method=method,
+        url_suffix=endpoint,
+        json_data=data
+    )
+    if response:
+        scan_response = response.get("scans")
+        markdown = tableToMarkdown(
+            'This is required Scan Result',
+            scan_response
+        )
+
+        result = CommandResults(
+            readable_output=markdown,
+            outputs_prefix='Rapid7.InsighVMCloud.Scans',
+            outputs_key_field='id',
+            outputs=scan_response
+        )
+        return result
+    else:
+        return_error(f'no response')
+
+
+def last_sites_command(client: Client, page: int, size=int) -> CommandResults:
+    """
+    Returns the details for sites.
+
+    Args:
+        client (Client Object): Client object with request.
+        page: The index of the page (zero-based) to retrieve
+        size: The number of records per page to retrieve.
+
+    """
+
+    args = demisto.args()
+    params = {
+        "page": args.get("page"),
+        "size": args.get("size")
+    }
+
+    if int(args.get("size")) > 500:
+        return_error("Exceed size limit")
+
+    else:
+        method = "POST"
+        endpoint = "/v4/integration/sites"
+        headers = ["name", "type"]
+        response = client.make_request(
+            method=method,
+            url_suffix=endpoint,
+            params=params
+        )
+        if response:
+            res = response.get("data")
+            markdown = tableToMarkdown('List Sites', res, headers)
+            result = CommandResults(
+                readable_output=markdown,
+                outputs_prefix='Rapid7.InsighVMCloud.Sites',
+                outputs_key_field='name',
+                outputs=response
+            )
+            return result
+        else:
+            return_error("no response")
+
+
+def search_vulnerabilities_command(client: Client, query: str, page: int, size=int) -> dict:
```

Comment:
Why do we need the variables `age: int, size: int` they are not used ? @thimanshu474

Reply:
Its is used as a params in the command in line 360

Reply:
You use strings, not variables, so in that case the variables are unused

Reply:
To resolve the issue please edit like this:
    page = args.get("page")
    size = args.get("size")
    query = args.get("query")
    params ={
        "page": page,
        "size": size
    }

Reply:
It won't solver it the variables in the function are still redundant page for example will be `args.get("page")` if it exist or `None` if it's not and the page in the function signature is meaningless `search_vulnerabilities_command(client: Client, query: str, page: int, size=int)` why do you need the variables in the signature if you're getting them from args ?
@thimanshu474

Reply:
Please delete the variable taken from args and replace those line with below code in which directly taken value from function signature variables.

def search_vulnerabilities_command(client: Client, query: str, page: str, size: str)-> dict:

    params ={
        "page": page,
        "size": size
    }
    if int(size) > 500:
        return_error("Exceed size limit")